bookkeeper-server/src/main/java/org/apache/bookkeeper/auth/BookieAuthProvider.java

```diff
@@ -79,16 +75,19 @@ default void close() {}
     }
 
     /**
+     * Callback to let the provider know that the underlying protocol is changed.
```

Comment:
Is there a specific action that is needed as part of this callback in AuthProvider?